Projects/Server/main.cpp

```diff
@@ -8,19 +9,18 @@
 
 int main()
 {
-    GameHandler gameHandler{};
-
     //AbstractPlayer::Ptr player0 = std::make_shared<StackAI::StackPlayer>();
-    //AbstractPlayer::Ptr player1 = std::make_shared<TranqAI::TranqPlayer>();
+    AbstractPlayer::Ptr player1 = std::make_shared<TranqAI::TranqPlayer>();
 
     //AbstractPlayer::Ptr player0 = std::make_shared<RandomPlayer>();
     AbstractPlayer::Ptr player0 = std::make_shared<InteractivePlayer>();
-    AbstractPlayer::Ptr player1 = std::make_shared<RandomPlayer>();
-
-    gameHandler.init(player0, player1);
-
-    //for (size_t repeat = 0; repeat < 10; ++repeat)
-        gameHandler.play();
-
+    //AbstractPlayer::Ptr player1 = std::make_shared<RandomPlayer>();
```

Comment:
Exceptionally, I'm fine with leaving comments in the code, but can we please seperate `player0` and `player1˙ lines?

Reply:
They are adjusted so that, `player0` and `player1` are grouped with their own.